src/CmdLine/Type.hs

```diff
@@ -18,18 +18,18 @@ isWebCmdLine _ = False
 
 data CmdLine
     = Search
-        {color :: Bool
-        ,link :: Bool
-        ,info :: Bool
-        ,exact :: Bool
-        ,databases :: [FilePath]
-        ,start :: Maybe Int
-        ,count :: Maybe Int
-        ,web :: Maybe String
-        ,repeat_ :: Int
-        ,queryChunks :: [String]
-        ,queryParsed :: Either ParseError Query
-        ,queryText :: String
+        { color :: Bool
+        , link :: Bool
+        , info :: Bool
+        , exact :: Bool
+        , databases :: [FilePath]
+        , start :: Maybe Int
+        , count :: Maybe Int
+        , web :: Maybe String
+        , repeat_ :: Int
+        , queryChunks :: [String]
+        , queryParsed :: Either ParseError Query
+        , queryText :: String
```

Comment:
Ah, fair enough, I guess you are being tripped up by my past inconsistencies - can't ask for more than keeping the style consistent, whatever it might be!